CHANGES.md

```diff
@@ -36,6 +36,15 @@
   [gh-532]: https://github.com/jrblevin/markdown-mode/issues/532
   [gh-534]: https://github.com/jrblevin/markdown-mode/issues/534
 
+*Sep 02, 2020*
+
+*   Improvements:
+
+    -   Option to define a `markdown-link-make-text-function` function
```

Comment:
Sorry I didn't explain it clearly. Add two lists at `New features` section. And the data(`*Sep 02, 2020*`) is not necessary, because it is release date and version 2.5 is not released yet.

Reply:
Ah I see, that makes sense. Changes should be grouped per 'stable release' version right? Please see if my latest commit fixes it.

Reply:
I think your change introduces new features so I suppose they should be added at `New features` section as below image. How about ?

![tmp](https://user-images.githubusercontent.com/554281/92112786-3f392180-ee29-11ea-8f39-b40f8ea60d2a.png)

Reply:
My bad, I inserted the changes thinking they were in the new features section, sorry for the extra work.
Pushed a new commit now.